apifuzzer/swagger_template_generator.py

```diff
@@ -26,8 +27,8 @@ def normalize_url(url_in):
         # we will change back later
         return url_in.strip('/').replace('/', '+')
 
-    def process_api_resources(self):
-        self.logger.info('Start preparation')
+    def strategy_default():
+        self.logger.info('Start preparation with default strategy')
```

Comment:
It should be the default strategy, no need to introduce new config parameter and duplicate the code

Reply:
fixed in latest commits